cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -390,12 +477,22 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
     String gcpCmekKeyName = getString(properties, "encryptionKeyName");
     String gcpCmekBucket = getString(properties, "gcsBucket");
 
+    String clustermetadata = getString(properties, "clustermetadata");
+    String networktags = getString(properties, "networktags");
+    String initactions = getString(properties, "initactions");
+    boolean enableautoscaling = Boolean.parseBoolean(properties.getOrDefault("enableautoscaling",
+            "true"));
```

Comment:
Should use `Boolean.toString(true)`.

Reply:
Did not get this . I am trying to retrieve a Boolean object.

Reply:
Instead of use the literal `"true"`, use `Boolean.toString(true)`, as this prevent future mistake in mis-spelling.